src/node/util.ts

```diff
@@ -157,12 +157,7 @@ export const generatePassword = async (length = 24): Promise<string> => {
  * Used to hash the password.
  */
 export const hash = async (password: string): Promise<string> => {
-  try {
-    return await argon2.hash(password)
-  } catch (error: any) {
-    logger.error(error)
-    return ""
-  }
```

Comment:
✍🏼 as discussed with @code-asher, this will only throw if you throw in something other than `string` (like `number`). Since TS will prevent that, we don't need the `try/catch` block.